src/RolloutScorer/RolloutScorer/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.Azure.KeyVault.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+using Octokit;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Reflection;
+
+namespace RolloutScorer
+{
+    public class Utilities
+    {
+        public const string HotfixAzureDevOpsTag = "[HOTFIX]";
```

Comment:
A big class full of unrelated constants isn't of great value.

If we must use constants (and I usually don't like them unless it's used in multiple methods, and then if it's inside the same class, they should be private in that class), they need to go into usefully named classes.

Like it should be GitFileMode.FileBlob = "100644", and LabelNames.Issue = "Rollout Issue".

Reply:
I like this idea. Swapping to that.